src/dev/src/app/tabs/tabs-static.demo.html

```diff
@@ -3,45 +3,94 @@
   ~ This software is released under MIT license.
   ~ The full license information can be found in LICENSE in the root directory of this project.
   -->
+<p>You cannot switch between tabs in this static demo because it does not contain the JavaScript to handle this behavior. The
+  demo is here to show the look of tabs and the DOM structure.</p>
 
 <div class="clr-example">
-  <ul id="demoTabs" class="nav" role="tablist">
-    <li role="tab" class="nav-item">
-      <button id="tabDashboard" class="btn btn-link nav-link active" aria-controls="panelDashboard" aria-selected="true" type="button">Dashboard</button>
-    </li>
-    <li role="tab" class="nav-item">
-      <button id="tabManagement" class="btn btn-link nav-link" aria-controls="panelManagement" aria-selected="false" type="button">Management</button>
-    </li>
-    <li role="tab" class="nav-item">
-      <button id="tabCloud" class="btn btn-link nav-link" aria-controls="panelCloud" aria-selected="false" type="button">Cloud</button>
-    </li>
-    <div class="tabs-overflow open bottom-right">
+  <h3>Static tabs</h3>
+  <div class="demo">
+    <ul id="demoTabs" class="nav" role="tablist">
       <li role="tab" class="nav-item">
-        <button class="btn btn-link nav-link dropdown-toggle">
-          <clr-icon shape="ellipsis-horizontal" class="is-info"></clr-icon>
-        </button>
+        <button id="tabDashboard" class="btn btn-link nav-link active" aria-controls="panelDashboard" aria-selected="true" type="button">Dashboard</button>
       </li>
-      <div class="dropdown-menu">
-        <button class="btn" aria-controls="panelInfrastructure">Infrastructure</button>
-        <button class="btn" aria-controls="panelSettings">Settings</button>
+      <li role="tab" class="nav-item">
+        <button id="tabManagement" class="btn btn-link nav-link" aria-controls="panelManagement" aria-selected="false" type="button">Management</button>
+      </li>
+      <li role="tab" class="nav-item">
+        <button id="tabCloud" class="btn btn-link nav-link" aria-controls="panelCloud" aria-selected="false" type="button">Cloud</button>
+      </li>
+      <div class="tabs-overflow open bottom-right">
+        <li class="nav-item">
```

Comment:
I know its a demo but shouldn't this button have role="tab" still?

Reply:
It's negotiable. It's both a tab and a menu item. I'd be happy to get a bit more discussion around the accessibility of the overflow. Maybe involve the new a11y engineer once he joins the team?

Reply:
Looking more closely at it, this is only the [...] overflow activation button.
It has not role of a tab, it only opens the context menu, where the buttons do have the role=tab attribute. See lines 29, 30 bellow.